packages/remix-dev/cli/migrate/migrations/convert-to-javascript/transform/createExportExpressionStatement.ts

```diff
@@ -0,0 +1,71 @@
+import type { ExportDefaultDeclaration, JSCodeshift } from "jscodeshift";
+
+export const createExportExpressionStatement = (
+  j: JSCodeshift,
+  exportDefaultDeclaration: ExportDefaultDeclaration
+) => {
+  /**
+   * HACK: Can't use casts nor type guards in a `jscodeshift` transform
```

Comment:
We know for sure that `exportDefaultDeclaration.declaration` isn't of `DeclarationKind`, but there's unfortunately no other way than checking each possible union value separately, since we can't cast in a `jscodeshift` transform

Reply:
Is this just to make TypeScript happy with this code or is it actually needed at runtime?

Reply:
The whole hack is to make TS happy 😢

Reply:
Found a way to convert these as well